webservice/template/admin/assignment/update_new.tmpl

```diff
@@ -0,0 +1,203 @@
+                    <button type="button" id="toggleReview" class="btn btn-danger btn-sm mb-3">Disable</button>
+
+                    <div class="form-group">
+                        <label for="review_deadline">Review Deadline</label>
+                        <input type="datetime-local" name="review_deadline" id="review_deadline" class="form-control" value="{{.ReviewDeadline}}">
+                    </div>
+
+                    <hr>
+
+                    <h5>Required reviews done</h5>
+                    <div class="form-group">
+                        <label for="reviewers">How many submissions is required for each student to review on this assignment.</label>
+                        <select name="reviewers" id="reviewers" class="form-control" required>
+                            {{if .Assignment.Reviewers.Valid}}
+                                <option value="{{.Assignment.Reviewers.Int64}}" selected>{{.Assignment.Reviewers.Int64}}</option>
+                            {{end}}
+                            <option value="0">Any</option>
+                            <script>
+                                for (let i = 1; i <= 6; i++) {
+                                    document.write(`<option value="${i}">${i}</option>`);
+                                }
+                            </script>
+                        </select>
+                        <small class="form-text text-muted">Set to <code>Any</code> for no requirements.</small>
+                    </div>
+                </div>
+
+            </div>
+        </div>
+
+        <div class="row mb-5">
+            <div class="col">
+                <input type="hidden" name="id" value="{{.Assignment.ID}}">
+                <input type="hidden" id="review_enabled" name="review_enabled" value="{{if .Assignment.ReviewID.Valid}}true{{else}}false{{end}}">
+                <input type="submit" class="btn btn-success btn-lg btn-block" id="submit">
+            </div>
+        </div>
+
+    </form>
+{{end}}
+{{define "foot"}}
+    <script src="/static/js/simplemde.min.js"></script>
+    <script src="/static/js/time.js"></script>
+    <script>
+        (function () {
+            // Enable the SmpleMDE
+            let simplemde = new SimpleMDE({
+                element: document.getElementById('simplemde'),
+                hideIcons: ["side-by-side", "fullscreen"],
+                lineWrapping: true,
+            });
+
+            let reviewSelect = document.getElementById('review_id');
+            let reviewDeadlineContainer = document.getElementById('review_deadline_container');
+            let reviewDeadline = document.getElementById('review_deadline');
+            let toggleReviewBtn = document.getElementById('toggleReview');
+
+            toggleReviewBtn.addEventListener('click', () => {
+                reviewSelect.selectedIndex = 0;
+                reviewDeadlineContainer.classList.add('sr-only');
```

Comment:
don't hide the review stuff even if it is disabled. It should be enabled/disabled by the button